docs/testing/e2e-plan.md

```diff
@@ -0,0 +1,73 @@
+
+#### Investigate Daily E2E Run
+
+Running end-to-end tests more frequently is another priority of the short term plan. An investigation into the feasibility of adding a periodic run of the end-to-end tests will be performed. If a periodic end-to-end run is estimated to be within a few days' work, then it will be undertaken. The intention is to be notified of end-to-end test failures before a release branch is created, when deadlines are already set and there are possibly many time-sensitive tasks already waiting to be completed.
+
+### Long Term
+
+Following the short term goals, several broader, more in-depth changes will help our end-to-end tests further raise our confidence. What follows is a brief summary of the larger components or actions required.
+
+A more detailed design will be submitted for review after the short term priorities have been satisfied.
+
+#### Test Manifests
+
+A list of each artifact produced, along with identifiers and URLs to acquire each artifact for testing. This manifest will serve as one input to a new testing framework that will be able to ensure that the desired combinations of providers, host OSes, and any other test dimensions deemed appropriate are run.
+
+This manifest will be generated from a commit to the eks-anywhere-packages repository, and will in turn be used by the testing framework.
+
+#### Increased Frequency Of E2E Test Runs
+
+If feasible, end-to-end tests will be run on each (ok-to-test) pull request on the eks-anywhere-packages repository. If that isn't feasible, then hourly or semi-hourly test runs will help catch problems sooner.
+
+#### Test Configuration Control
+
+A means by which tests can be disabled or enabled based upon their properties, requirements, or current statuses. In plainer words, a way to write a test that doesn't run for a specific provider, or to temporarily disable tests for specific host OS. This input, along with the test manifest are envisioned as being the bulk of the input to a given CP E2E test run.
+
+#### End-To-End Test Documentation
+
+Documentation will be generated that, in addition to the forthcoming design document, will aid developers and other contributors understand the end-to-end test framework, specifically how to write a new end-to-end test, how to run an end-to-end test outside of the build pipelines, how to be notified of failures, and how to debug test failures.
```

Comment:
I'd add a stretch goal of adding documentation tests. Similar to https://doc.rust-lang.org/rustdoc/write-documentation/documentation-tests.html

Reply:
...or? https://pkg.go.dev/testing#hdr-Examples